<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: Open Sans SemiBold;
	font-color: white;
}


QPushButton{
    background-color: #2e9cca;
    border-style: solid;
    border-width: 2px;
    border-radius: 10px;
    border-color: #aaabb8;
	color: white;
	font-size: 12px;

}

QPushButton:pressed {
	border-width: 5px;
	border-color: white;
	
}

QPushButton:hover{
	background-color: #2964BA;
}

QPushButton:disabled{
	background-color: #464866
}

#centralwidget{
	background: #2A1B3D;
}

QLineEdit{
	background-color: #aaabb8;
	border-style: solid;
	border-width: 2px;
	border-radius: 10px;
	border-color: white;
	selection-background-color: dark grey;
	color: black;
	padding: 2px;
    font: bold;
	font-size: 16px;
	text-align: centre;

}

QLabel{
	color: white;
	font: bold;

}

QTableWidget{
	background-color: #464866;
	color: white;
	selection-background-color: #2964BA;
	border-color: #aaabb8;
	border-style: solid;
	border-width: 1px;
	border-radius: 15px;
	gridline-color: #aaabb8;
}

QRadioButton{
	color: white;
}

QComboBox{
	background: #464866;
	color: white;
	border-color: #aaabb8;
	border-style: solid;
	border-width: 3px;
	border-radius: 10px;
	font-size: 16px;
}

QComboBox:hover{
	background: #2964BA
}



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="q_table">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans SemiBold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>My Quizzes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans SemiBold</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="q_quiz_code">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="q_calculator">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>380</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans SemiBold</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Calculator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_create_quiz">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Quiz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_practice_quiz">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Practice Quiz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_load_quiz">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>280</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Quiz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_delete_quiz">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Quiz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_signout">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Out</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
